File: 2107-find-unique-binary-string/find-unique-binary-string.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        s = set()
        for num in nums:
            s.add(int(num, 2))
        for i in range(len(nums)+1):
            if i not in s:
                return '{0:{fill}{size}b}'.format(i, fill='0', size=len(nums))
        return ''

File: 2107-find-unique-binary-string/find-unique-binary-string.js
/**
 * @param {string[]} nums
 * @return {string}
 */
var findDifferentBinaryString = function(nums) {
    let s = new Set();
    let ans = "";
    for(let num of nums) {
        s.add(parseInt(num, 2));
    }
    for(let i=0; i<=nums.length; i++) {
        if(!s.has(i)) {
            ans = i.toString(2);
            ans = ans.padStart(nums.length, '0');
            return ans;
        }
    }
    return ans;
};